packages/wdio-cli/src/templates/wdio.conf.tpl.ejs

```diff
@@ -24,16 +24,6 @@ exports.config = {
     path: '<%= answers.path %>',<% } else if (answers.services.includes('appium')) { %>
     port: 4723,<% }
 
-    if(answers.services.includes('chromedriver')) { %>
```

Comment:
Do you think it makes sense now to set default path to wd/hub if selenium standalone is used? At least until v4 is out (no idea when is it going to happen)